### PR TITLE
Step2-Acc-27-company

### PR DESCRIPTION
**Pull request by AbbosB12**
Please review

### CODE REVIEW

File: src/main/java/com/example/accountingapp/controller/CompanyController.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        return "/company/company-edit";
+    }
+    @PostMapping("/update")
+    public String updateCompany(@ModelAttribute("company")CompanyDTO company,Model model){
+        companyService.update(company);
+        model.addAttribute("companies", companyService.listAllCompanies());
+        model.addAttribute("states", State.values());
+        model.addAttribute("status", CompanyStatus.values());
+        return "/company/company-list";
+    }
```
Comment (BahaMaksudov) [inline src/main/java/com/example/accountingapp/controller/CompanyController.java:78]: Edit Company State functionality is not working. This is part of CRUD and maybe will be worked on later. Please confirm. Other than that everything looks fine and works great. Thank you.

**Review by BahaMaksudov — COMMENTED** [review]
Added a comment regarding Edit Company functionality. Please check and confirm.

**Review by MehmetKaldirim — APPROVED** [review]
list work properly  good job, just next time please merge master before create a pull request ...

**Review by AccountingApp-Group2 — DISMISSED** [review]
phone does not show up, please fix before merging

image


image

**Comment by VitalySlabada** [comment]
I left the comment about the phone :) was signed in in different account - I think it just needs some data in sql to be added

**Review by MehmetKaldirim — APPROVED** [review]
There are three  bugs for edit , establishment date,status  and states dont change after update, u re taking data from db and save it same into db , if u have time please try to fix them...

**Comment by VitalySlabada** [comment]
@MehmetKaldirim 
I think Abbos is still on step 2
No need to do crud, only list properly

**Comment by josephum** [comment]
IdeaProjects/AccountingApp/src/main/java/com/example/accountingapp/controller/CompanyController.java:45:23
java: cannot find symbol
  symbol:   method save(com.example.accountingapp.dto.CompanyDTO)
  location: variable companyService of type com.example.accountingapp.service.CompanyService

Some CRUD methods like save are not implemented

**Comment by AbbosB12** [comment]
Crud Operation work properly

**Review by MehmetKaldirim — APPROVED** [review]

**Review by josephum — APPROVED** [review]

**Pull request merged**